Revert/Sources/AutoLayoutMarginsViewController.swift

```diff
@@ -22,22 +21,21 @@ final class AutoLayoutMarginsViewController: RevertViewController {
     }
   }
 
-  // MARK: Private
+  // MARK: MarginsAdjustingViewDelegate
+  func didUpdateMargins(updatedLayoutMargins: UIEdgeInsets) {
+    self.centerView.layoutMargins = updatedLayoutMargins
+  }
 
+  // MARK: Private
   private static let interSquareSpacing: CGFloat = 20
 
   private var lastUpdateSquaresWidthSize: CGSize?
 
-  @IBOutlet private weak var centerView: UIView!
-  @IBOutlet private weak var slider: UISlider!
-  @IBOutlet private weak var centerViewWidthConstraint: NSLayoutConstraint!
-  @IBOutlet private weak var containerView: UIView!
-  @IBOutlet private weak var containerViewBottomConstraint: NSLayoutConstraint!
-
-  @IBAction private func sliderValueChanged(sender: UISlider) {
-    let margin = CGFloat(sender.value)
-    self.centerView.layoutMargins = UIEdgeInsets(top: margin, left: margin, bottom: margin, right: margin)
-  }
+  @IBOutlet private var marginsAdjustingView: MarginsAdjustingView!
```

Comment:
Considering the other attribute's semantics I'd consider `adjustingViewMargins` rather.